docs/source/reference/respawn/clientcommands.csv

```diff
@@ -8,4 +9,11 @@ VModEnable; number; unknown, logged with number 0
 vban; number;unknown, logged with number 0
 HoldToRodeo; number0;unknown, logged with number 0
 ClientStatus; number;unknown, logged with number 0
-AllDialogueFinished;;unknown
\ No newline at end of file
+AllDialogueFinished;;unknown
+Map; <map> ex. mp_glitch; used to change the map in private matches 
+give; weapon or item name; give the player a weapon or item
+disconnect;; sends client to main menu
+ent_create; <entity type>; creates a entity at crosshair
+ent_throw; <entity type>; throws a entity
+ent_info; <entity type>; gives info on an entity
+bind; key command you want it to execute; binds a key to any console command. ex. `bind m "sv_cheats 1"`
```

Comment:
You appear to have confused ClientCommand for ConCommands. These are two different things, and concommands do not belong on this page.